EIPS/eip-proxy_ownership_register.md

```diff
@@ -0,0 +1,127 @@
+
+    Contract / Dapp                        Register
+
+      |                                       |
+      |------------- 0x0222..---------------> |
+      |                                       |
+      | <-------nominator: 0x0222..---------- |
+      |         delivery: 0x0222..            |
+      |                                       |
+
+If the EPS register is passed the address of a Nominator it will revert. This is of vital importance. The purpose of the proxy is that the Proxy address is operating on behalf of the Nominator. The Proxy address therefore can derive the same benefits as the Nominator (for example discord roles based on the Nominator's holdings, or mint NFTs that require another NFT to be held). It is therefore imperative that the Nominator in an active proxy cannot also interact and derive these benefits, otherwise two addresses represent the same holding. A Nominator can of course delete the Proxy Record at any time and interact on it's own behalf, with the Proxy address instantly losing any benefits associated with the proxy relationship.
+
+Full technical information is at https://docs.epsproxy.com/. 
+
+## Rationale
+
+The rationale for this design was to provide an easy and convenient way to prove ownership of assets while putting those assets at absolutely zero risk of loss. The motivation was seeing a number of attacks making use of the fact that users need to interact with a wallet holding assets in order to derive benefits of that asset ownership.
+
+In addition to the loss of user's assets it is clear that these events damage confidence in the community in general. Users make decisions based on both actual and perceived risk, and it presents a barrier for new users accessing and using ethereum.
+
+My vision is an ethereum where users setup a new hardware wallet for assets they wish to hold long-term, then make one single contract interaction with that wallet: to nominate a hot wallet proxy. That user can always prove they own assets on that address, and they can specify it as a delivery address for new asset delivery.
+
+## Backwards Compatibility
+
+The eip is fully backwards compatible.
+
+## Test Cases
+
+The full SDLC for this proposal has been completed and it is operation at 0xfa3D2d059E9c0d348dB185B32581ded8E8243924 on mainnet, ropsten and rinkeby. The contract source code is validated and available on etherscan. The full unit test suite is available in `../assets/eip-proxy_ownership_register/`, as is the source code and example implementaitons.
+
+## Reference Implementation
+
+The register is deployed at 0xfa3D2d059E9c0d348dB185B32581ded8E8243924 on mainnet, ropsten and rinkeby. The UI is available as follows:
+* Mainnet: https://app.epsproxy.com/
+* Ropsten: https://app-ropsten.epsproxy.com/
+* Rinkeby: https://app-rinkeby.epsproxy.com/
```

Comment:
Reference implementation should either be inline, included in `../assets/eip-4886/*` or the section should be left out entirely.  As mentioned above, external links should not be included in EIPs (they are just technical specifications).

Reply:
Good point, thank you! Can I leave the contract address in there without links to the web apps? Thank you!

Reply:
Unless it is something like EIP-1820 that is deployed to the same address on every EVM based blockchain, I recommend just including the contract contents as a reference implementation in `../assets/...` and not try to include deployment details (since for most things, that is unrelated to the standardization).

Reply:
Great idea, thank you. The reference implementation details are already in the assets folder.